test/controllers/RevisedUnloadingProcedureUnmetConditionsControllerSpec.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import base.{AppWithDefaultMockFixtures, SpecBase}
+import generators.Generators
+import models.UserAnswers
+import navigation.Navigation
+import org.mockito.ArgumentCaptor
+import org.mockito.Mockito.{reset, verify, verifyNoInteractions}
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.{JsObject, Json}
+import play.api.test.FakeRequest
+import play.api.test.Helpers.*
+import utils.transformers.IE043Transformer
+import views.html.RevisedUnloadingProcedureUnmetConditionsView
+
+import java.time.Instant
+
+class RevisedUnloadingProcedureUnmetConditionsControllerSpec extends SpecBase with AppWithDefaultMockFixtures with ScalaCheckPropertyChecks with Generators {
+
+  private lazy val revisedUnloadingProcedureUnmet: String = controllers.routes.RevisedUnloadingProcedureUnmetConditionsController.onPageLoad(arrivalId).url
+
+  private val mockTransformer = mock[IE043Transformer]
```

Comment:
I see that the controllers are still mocking these dependencies. Ideally we will only be mocking your new NewAuthYesNoSubmissionService